### ISSUE TITLE
Latest version of chrome not working to sync or load chat

### ISSUE THREAD

**Issue by dmandave** [JasonZigelbaum/jqbx-issues#171]
Win 10 with Chrome Version 74.0.3729.131 (Official Build) (64-bit) is not able to sync .

In console, getting these errors (Device ID replaced with XXX)::
VM19:1 PUT https://api.spotify.com/v1/me/player/play?device_id=XXX 404
(anonymous) @ VM19:1
f.end @ main.61d18dc7.js:106
o._makeRequest @ main.61d18dc7.js:105
o.put @ main.61d18dc7.js:105
a @ main.61d18dc7.js:105
_performRequest @ main.61d18dc7.js:105
play @ main.61d18dc7.js:105
(anonymous) @ main.61d18dc7.js:12
r.onmessage @ iwym:2
api.spotify.com/v1/me/player/seek?device_id=XXX&position_ms=119228:1 PUT https://api.spotify.com/v1/me/player/seek?device_id=XXX&position_ms=119228 404

**Comment by JasonZigelbaum** [comment]
Strange- can you do me a favor at let me know if
https://open.spotify.com/ works
for you or not also?

**Comment by dmandave** [comment]
yes, that https://open.spotify.com/ works, it opens without any issues

**Comment by momania** [comment]
I'm seeing similar issues since the latest Chrome version(s). Sync, etc, all works, but the JQBX webplayer won't play any audio in the Chrome tab.

On loading I see:
```javascript
main.398b8823.js:103 Uncaught TypeError: Cannot read property 'type' of undefined
    at main.398b8823.js:103
    at main.398b8823.js:104
    at Object.goHome (main.398b8823.js:104)
    at t.value (main.398b8823.js:10)
    at e.notifyAll (main.398b8823.js:51)
    at i.close (main.398b8823.js:101)
    at i.closeAll (main.398b8823.js:38)
    at i.perform (main.398b8823.js:38)
    at s (main.398b8823.js:51)
    at i.perform (main.398b8823.js:38)
    at Object.batchedUpdates (main.398b8823.js:51)
    at Object.o [as batchedUpdates] (main.398b8823.js:10)
    at Object._renderNewRootComponent (main.398b8823.js:51)
    at Object._renderSubtreeIntoContainer (main.398b8823.js:51)
    at render (main.398b8823.js:51)
    at Object.<anonymous> (main.398b8823.js:71)
```

**Comment by JasonZigelbaum** [comment]
Hey can you hit refresh and then try the web player. I pushed a fix
yesterday that should get the audio working. The issue was due to a Spotify
bug: https://github.com/spotify/web-playback-sdk/issues/75

**Comment by momania** [comment]
Awesome, works again. Thanks!
